commands/rm.go

```diff
@@ -34,10 +34,11 @@ func cmdRm(c *cli.Context) {
 		if err := mcn.Remove(host, force); err != nil {
 			log.Errorf("Error removing machine %s: %s", host, err)
 			isError = true
+		} else {
+			log.Infof("Successfully removed %s", host)
```

Comment:
I dunno. I like the `if isError` block. It reminds users that even though 99 of their 100 specified hosts were removed successfully, there _was_ one that had a problem. And it gives them some good hints on what to do about it.

+1 on porting this to `runActionWithContext`. Concurrency for the win!